src/lib/field-directive.ts

```diff
@@ -154,13 +156,19 @@ const createRateLimitDirective = (
             type: new GraphQLList(GraphQLString)
           },
           max: {
-            type: GraphQLInt
+            type: new GraphQLUnionType({
+              name: 'max',
+              types: [GraphQLInt, GraphQLString]
+            })
```

Comment:
`ms` isn't needed on this arg, right? `max` is not a time value

Reply:
Oh what was I doing, jeeeez 🤦‍♂️

Reply:
😂